### PR TITLE
PMM-12119 fix rds_exporter status update

### PR DESCRIPTION
**Pull request by atymchuk**
PMM-12119

Link to the Feature Build: SUBMODULES-3280

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

File: managed/services/agents/roster.go [hunk 1]
```diff
@@ -35,34 +39,62 @@ const (
...
+
+	sort.Strings(exporterIDs)
+
+	r.m[groupID] = exporterIDs
+	r.l.Debugf("add: %s = %v", groupID, exporterIDs)
+	return groupID
 }
 
-func (r *roster) get(groupID string) (agentIDs []string) { //nolint:nonamedreturns
+func (r *roster) get(groupID string) (string, []string, error) {
```
Comment (idoqo) [inline managed/services/agents/roster.go:72]: i know the linter frowns at named returns - maybe we can add method comments to describe what each return parameter is/does?

**Review by idoqo — APPROVED** [review]

File: managed/services/agents/roster.go (hunk 1 above)
Comment (atymchuk) [reply managed/services/agents/roster.go:75]: Sure. Done!

File: managed/services/agents/roster.go [hunk 2]
```diff
@@ -35,36 +39,68 @@ const (
...
 
-func (r *roster) get(groupID string) (agentIDs []string) { //nolint:nonamedreturns
+// get returns a PMMAgentID and a group of exporter IDs for a given Group ID.
+func (r *roster) get(groupID string) (string, []string, error) {
 	r.rw.RLock()
 	defer r.rw.RUnlock()
 
-	agentIDs = r.m[groupID]
+	suffix := "/" + string(rdsGroup)
+	PMMAgentID, ok := strings.CutSuffix(groupID, suffix)
```
Comment (rishat-ishbulatov) [inline managed/services/agents/roster.go:80]: I think this is not critical, a matter of taste, but it looks like we don’t need to allocate memory for a `suffix` every time and we could create it once using const at the compile time with `const suffix = "/" + string(rdsGroup)` [This](https://stackoverflow.com/questions/35026810/runtime-const-in-golang) place describes this minor tweak.

**Review by rishat-ishbulatov — APPROVED** [review]

File: managed/services/agents/roster.go (hunk 2 above)
Comment (atymchuk) [reply managed/services/agents/roster.go]: No problem, done!

**Pull request merged**